libs/wingc/src/parser.rs

```diff
@@ -890,14 +889,22 @@ impl<'s> Parser<'s> {
 			"function_type" => {
 				let param_type_list_node = type_node.child_by_field_name("parameter_types").unwrap();
 				let mut cursor = param_type_list_node.walk();
-				let param_types = param_type_list_node
-					.named_children(&mut cursor)
-					.filter_map(|param_type| self.build_type_annotation(&param_type, phase).ok())
-					.collect::<Vec<TypeAnnotation>>();
+
+				let mut parameters = vec![];
+				for param_type in param_type_list_node.named_children(&mut cursor) {
+					let t = self.build_type_annotation(&param_type, phase)?;
+
+					parameters.push(FunctionParameter {
+						name: "".into(),
```

Comment:
Usually when a value isn't always available, it's preferable to model it in Rust using the `Option` type, e.g. `Option<String>`